NNLOops.md

```diff
@@ -3,14 +3,27 @@
     <th><b>Description</b></th>
     <th><b>Code</b></th>
   </tr>
-  <tr>
-    <td>For every event, for every jet, take all tracks and find the closest track that has pt>1 gev, and then plot the delta-R of that track and jet, the jet's pT, and the tracks pT.</td>
-    <td>???</td>
-  </tr>
   <tr>
     <td>If the sum of the pt of all muons with eta greater than 1 is greater than 30, fill histogram with pt of all muons.</td>
     <td>
+      RDF
+      <pre lang="cpp">df.Filter("Sum(Muon_pt[abs(Muon_eta) > 1]) > 30").Histo1D("Muon_pt")</pre>

```

Comment:
should we add the `abs` in the `TTree::Draw` example too?

Reply:
done